src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -94,6 +94,13 @@ interface PostLogConfig = {
 To enable simulcast for the meeting pass the `simulcastEnabled` prop with value set to `true`. By default, it is `false`.
 For more information on Simulcast, check Amazon Chime JS SDK [simulcast guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/05_Simulcast.md).
 
+#### logger
+The `Logger` object that you want to be used in the meeting session.
+If you pass in a `Logger` object using this parameter, the `MeetingManager` will use this object instead of creating a logger based on `logLevel` to initialize the meeting session.
+
+#### videoDownlinkBandwidthPolicy
+The `VideoDownlinkBandwidthPolicy` object that you want to be used in the meeting session.
```

Comment:
Can add later, but since the React SDK does not have any guides on video downlink policy it would be good to link the JS SDK guides from here so the builders can get more information about it.

Reply:
Maybe we should include a code sample on how they can pass priority downlink policy for example

Reply:
Yeah agree, a how-to suits here.

Reply:
Good idea, will do!

Reply:
Updated